app-backend/migrations/src_migrations/main/scala/104.scala

```diff
@@ -0,0 +1,8 @@
+import slick.jdbc.PostgresProfile.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M104 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(104)(List(
+    sqlu"" // your sql code goes here
```

Comment:
Should there be some SQL here?

Reply:
...lol yes. i think i accidentally nuked it with an `rsync-back`

Reply:
I'm taking my [`nocode`](https://github.com/kelseyhightower/nocode) inclinations to new heights

Reply:
Very impressive.